dokka-subprojects/core/src/main/kotlin/org/jetbrains/dokka/links/DRI.kt

```diff
@@ -139,6 +167,11 @@ public data class PointingToCallableParameters(val parameterIndex: Int) : DriTar
     override fun toString(): String = "PointingToCallableParameters($parameterIndex)"
 }
 
+@ExperimentalDokkaApi
+public data class PointingToContextParameters(val parameterIndex: Int) : DriTarget() {
```

Comment:
This is used only for merging context parameters in [DefaultDocumentableMerger.kt](https://github.com/Kotlin/dokka/pull/4080/files#diff-00add2e0f152afbda9fba4e4e7e8d666bcfb619525d9d73bb211e7776e7bd1a9)